WordPress/Docs/NamingConventions/ValidPostTypeSlugStandard.xml

```diff
@@ -0,0 +1,117 @@
+<documentation title="Validate post type slugs">
+    <standard>
+    <![CDATA[
+    The post type slug used in register_post_type() can not be longer then 20 characters.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: short post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>'my_short_slug'</em>, 
+  array() 
+);
+        ]]>
+        </code>
+        <code title="Invalid: too long post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>'my_own_post_type_too_long'</em>, 
+  array() 
+);
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The post type slug used in register_post_type() can only contain lowercase alphanumeric characters, dashes and underscores.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: no special characters in post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>'my_post_type_slug'</em>, 
+  array() 
+);
+        ]]>
+        </code>
+        <code title="Invalid: invalid characters in post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>'my/post/type/slug'</em>, 
+  array() 
+);
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    One should be careful with passing dynamic slug names to "register_post_type()", as they may be too long and can contain invalid characters.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: static post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>'my_post_active'</em>, 
+  array() 
+);
+        ]]>
+        </code>
+        <code title="Invalid: dynamic post type slug.">
+        <![CDATA[
+register_post_type( 
+  <em>"my_post_{$status}"</em>, 
+  array() 
+);
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The post type slug used in register_post_type() can not use reserved keywords.
```

Comment:
I'm just wondering if "reserved keywords" should be expanded a bit to explain what that means ?